README_fa-IR.md

```diff
@@ -0,0 +1,477 @@
+#  راهنمای Commit-Message
+
+[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+راهنمایی برای درک اهمیت Commit-Message و نحوه نوشتن صحیح آن.
+
+ این راهنما سعی دارد به شما کمک کند یادبگیرید کامیت چیست، چرا نوشتن Commit-Message مناسب مهم است،
+تمرین های خوب و نکاتی که می تواند منجرب به بازنگری و داشتن تاریخچه کامیت بهتری شود
+
+## دیگر زبان ها
```

Comment:
There's no need to add this section to the translation, but don't worry, I'm going to remove it after merge.